src/tests/assert.ts

```diff
@@ -20,22 +20,22 @@ export module Assert {
 
 		// Assert tab index ordering
 		for (let i = 1; i < elements.length; i++) {
-			ok(elements[i].elem.tabIndex > elements[i - 1].elem.tabIndex,
-				"Element " + elements[i].name + " should have a greater tabIndex than element " + elements[i - 1].name);
-		}
+				ok(elements[i].elem.tabIndex > elements[i - 1].elem.tabIndex, "Element " + elements[i].name + " whose tabIndex is" + elements[i].elem.tabIndex + " should have a greater tabIndex than element " + elements[i - 1].name + " whose tabIndex is " + elements[i - 1].elem.tabIndex);
+			}
 	}
 
-	export function posInSetIsDecremental(elementIds: string[]) {
+	export function posInSetIsIncremental(elementIds: string[]) {
 		let elementsInExpectedPosInSetOrder = elementIds.map(id => {return{name: id, elem: document.getElementById(id) }; });
-		Assert.posInSetIsDecrementalForElements(elementsInExpectedPosInSetOrder);
+		Assert.posInSetIsIncrementalForElements(elementsInExpectedPosInSetOrder);
 	}
 
-	export function posInSetIsDecrementalForElements(elements: {name: string, elem: HTMLElement}[]) {
+	export function posInSetIsIncrementalForElements(elements: {name: string, elem: HTMLElement}[]) {
```

Comment:
Test name could be more descriptive. We're not just testing that posinset is incremental, but it follows a strict order of 1, 2 ... n